old-scala-codegen/BUILD

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library")
+
+scala_library(
+    name = "old-scala-codegen",
```

Comment:
Why is this `old`? It is still in use. Do you have a plan to replace it completely in the near future?

Reply:
yes, that's the plan